Maquetado/scss/details.scss

```diff
@@ -15,16 +15,37 @@
 }
 
 .books-container {
+  height: 100%;
+  width: 100%;
+}
+
+.back-button {
+  font-size: 20px;
+  line-height: 24px;
+  margin: 50px 30px;
+
+  &:before {
+    content: url('assets/back-arrow-20.png');
+    padding: 10px;
+    vertical-align: middle;
+  }
+}
+
+.container {
   background-color: $wild-sand;
   height: 100%;
   width: 100%;
 }
 
+.container-book {
+  display: flex;
```

Comment:
Using `flex` here is necessary?

Reply:
If i don't use it, then the elements are not aligned. Instead, is one above the other.